src/platform/ASR/PlatformManagerImpl.cpp

```diff
@@ -51,17 +51,12 @@ CHIP_ERROR PlatformManagerImpl::_InitChipStack(void)
     err = Internal::InitLwIPCoreLock();
     SuccessOrExit(err);
 
-    /* Initialize the event flags. */
-    result = lega_rtos_init_event_flags(&mEventFlags);
```

Comment:
These changes to how timers are handled don't obviously seem related to the locking bits, but ok....